### PR TITLE
[IMP] web: implement field for many2one_reference fields

### PR DESCRIPTION
**Pull request by aab-odoo**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/web/models/models.py [hunk 1]
```diff
@@ -189,7 +189,10 @@ def cleanup(vals: Dict) -> Dict:
                         co_record = co_record.with_context(**field_spec['context'])
 
                     if 'fields' in field_spec:
-                        reference_read = co_record.web_read(field_spec['fields'])
+                        try:
+                            reference_read = co_record.web_read(field_spec['fields'])
+                        except AccessError:
+                            reference_read = [{'id': co_record.id, 'display_name': '_NoAccess_'}]
```
Comment (seb-odoo) [inline addons/web/models/models.py:195]: I don't know if this was discussed but maybe we'd want this value to be translated?

**Review by kebeclibre — APPROVED** [review]
lgtm

**Review by ryv-odoo — APPROVED** [review]
LGTM for the python part.

**Comment by aab-odoo** [comment]
Hello @odoo/rd-security 

Could you check the CI/security error raised on this PR? I added a model_field attribute on Many2oneReference fields, such that the JS implementation of that fields knows in which field the model is encoded.

**Comment by kebeclibre** [comment]
@robodoo rebase-ff r+

**Pull request closed without merging**